File: api/v2010/address/create-example/create-example.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Api.V2010.Account;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var address = AddressResource.Create(
            customerName: "Customer 500",
            street: "Elm Street",
            city: "Racoon",
            region: "Mordor",
            postalCode: "150",
            isoCountry: "AX"
        );

        Console.WriteLine(address.Sid);
    }
}

File: api/v2010/address/create-example/create-example.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

address = @client.addresses.create(
                              customer_name: 'Customer 500',
                              street: 'Elm Street',
                              city: 'Racoon',
                              region: 'Mordor',
                              postal_code: '150',
                              iso_country: 'AX'
                            )

puts address.sid
